packages/react-ui/src/app/features/builder/blocks-selector/block-selector-utils.ts

```diff
@@ -46,11 +47,13 @@ const getStepName = (block: StepMetadata, flowVersion: FlowVersion) => {
 };
 
 const getDefaultStep = ({
+  id,
   stepName,
   block,
   actionOrTriggerName,
   displayName,
 }: {
+  id: string;
```

Comment:
maybe I'm blind but I don't see where you actually pass null/undefined to force the new id generation

Reply:
fixed